CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Add the possibility to plot and update frames in the Matlab visualizer.
+- Added ``getFileLocationOnLocalFileSystem`` method in ``ExternalMesh`` that attempt to find the mesh location in the local file system
```

Comment:
Can we specify that this is now used when using the visualizer?

Reply:
Done in https://github.com/robotology/idyntree/pull/798/commits/a98f5ef59f64034542efea28c336d2f7b6be05d5